lib/src/Base/Graph/Cloud.cxx

```diff
@@ -112,6 +112,22 @@ String Cloud::__repr__() const
   return oss;
 }
 
+/* Accessor for boundingbox */
+Cloud::BoundingBox Cloud::getBoundingBox() const
+{
+  BoundingBox boundingBox(BoundingBoxSize);
+  const Point max(data_.getMax());
+  const Point min(data_.getMin());
+  const Scalar x_margin = ResourceMap::GetAsScalar("Cloud-HorizontalMargin");
+  const Scalar y_margin = ResourceMap::GetAsScalar("Cloud-VerticalMargin");
+  const Point delta(max - min);
```

Comment:
xMargin/yMargin (see OT coding rules)
Btw we should check that values are not negative